agent/app/agent_capability.go

```diff
@@ -61,29 +64,45 @@ const (
 	capabilityEFS                               = "efs"
 	capabilityEFSAuth                           = "efsAuth"
 	capabilityEnvFilesS3                        = "env-files.s3"
+	capabilityExec                              = "execute-command"
+	capabilityDepsRootDir                       = "/managed-agents"
```

Comment:
Should it have ${VERSION} folder of ssm-agent as well?

Reply:
yup, the version sub folder was added after this PR was posted, adding in the next iteration